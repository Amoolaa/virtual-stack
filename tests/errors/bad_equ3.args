"tests/errors/bad_equ3.x2017"
