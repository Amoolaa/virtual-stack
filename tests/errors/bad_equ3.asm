FUNC LABEL 0
    EQU VAL 9
    EQU VAL 0
    RET
