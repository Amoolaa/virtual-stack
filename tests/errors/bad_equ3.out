EQU: argument must be register.
